packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx

```diff
@@ -101,6 +101,9 @@ function StrictModeEnablement(props: TStrictModeEnablementProps) {
   }
 }
 
+const isLocalProdMode =
```

Comment:
**nit**: Maybe we can leave a comment here explaining why we need this constant.